app/controllers/api/v1/book_suggestion_controller.rb

```diff
@@ -2,7 +2,8 @@ module Api
   module V1
     class BookSuggestionController < ApplicationController
       def create
-        return user_not_found if params[:user_id].present? && !User.exists?(params[:user_id]) 
+        return user_not_found if params[:user_id].present? && !User.exists?(params[:user_id])
```

Comment:
Wouldn't this be an OR? Or is the user_id param not mandatory?

Reply:
the user_id parameter is not mandatory, because as it is not present it is considered as a guest user